File: laboratorios/lab02/l-javascript/clases/Profesor.js
import { Persona } from "./Persona.js"

export class Profesor extends Persona {
  constructor(id, nombre, edad) {
    super(id, nombre, edad)
  }

  toString() {
    return `Profesor{id='${this.id}', nombre='${this.nombre}', edad=${this.edad}}`
  }
}

File: laboratorios/lab02/l-java/src/main/java/com/tuorg/poo/clases/Profesor.java
package com.tuorg.poo.clases;

//Hereda de Persona
public class Profesor extends Persona {
    public Profesor(String id, String nombre, int edad) {
        super(id, nombre, edad);
    }

    @Override
    public String toString() {
        return "Profesor{" +
                "id='" + getId() + '\'' +
                ", nombre='" + getNombre() + '\'' +
                ", edad=" + getEdad() +
                '}';
    }
}
